jenkins/swift-functional-tests/20-install-openstack.sh

```diff
@@ -30,11 +48,21 @@ function centos_specifics {
     sudo yum install -y wget
     wget https://bootstrap.pypa.io/ez_setup.py -O - | sudo python;
     sudo easy_install -U six
+    sudo yum install -y python-pip
+    if [[ $DEVSTACK_BRANCH == "stable/icehouse" ]]; then
+        # Required to get 'cryptography' python package compiled during its installation through pip.
+        sudo yum install -y gcc python-devel libffi-devel openssl-devel
+        # Required by keystoneclient
+        sudo yum install -y MySQL-python
```

Comment:
The comment about which openstack package or dependency need those os-packages shall be useful for future devstack installation headache, so I'd like to keep it that way.